src/app/collective-rewards/allocations/page.tsx

```diff
@@ -77,13 +83,13 @@ export default function Allocations() {
         </div>
         <div className="flex flex-col items-start gap-6 self-stretch">
           <AllocationMetrics />
-          <AllocationAmount />
+          <AllocationAmount resetTriggered={resetTrigger} />
```

Comment:
I know, I know, we're in hurry, but to reset a react component in an idiomatic way is to change it's key, not to pass a reset function.
```suggestion
          <AllocationAmount key={resetCounter} />
```